src/vm/analysis/type_checker/tests/mod.rs

```diff
@@ -104,18 +104,18 @@ fn test_simple_arithmetic_checks() {
 
 #[test]
 fn test_simple_hash_checks() {
-    let good = ["(hash160 1)",
-                "(sha512 10)",
-                "(sha512/256 10)",
-                "(sha256 (keccak256 1))"];
+    let good = ["(hash160 u1)",
```

Comment:
Can you preserve the tests with both signed and unsigned integers here?

Reply:
Addressed with https://github.com/blockstack/blockstack-core/pull/1129/commits/50149b7a553118e70688d3cd2ea47e9e516683c4